src/compilers/try.cr

```diff
@@ -1,36 +1,42 @@
 module Mint
   class Compiler
+    protected def prefix(_node : Ast::Try, statement : Ast::Statement, value : String)
```

Comment:
Isn't this the same as for `sequence` and `parallel`?

Reply:
It is indeed - for `sequence` and `try`, although I wanted to maintain a clear separation of their definitions - where would you declare the shared method - in `src/compilers/try.cr` or `src/compilers/sequence.cr`? It could live in `src/compiler.cr` but that would be far away from its actual usage...

Reply:
The only difference is between `sequence`/`try` and `parallel` is a `const` keyword, so it would make sense to me to have only one implementation with a flag. 

That said it can stay as is for now, I'm planning to merge all of them into a single feature in the future anyway.